code/datums/components/tackle.dm

```diff
@@ -0,0 +1,338 @@
+			wearing a helmet should be enough to remove any chance of permanently paralyzing yourself and dramatically lessen knocking yourself unconscious, even with rocket gloves. Will expand on maybe
+
+		Wearing a helmet: -5
+		Clumsy: +6
+
+	Effects: Below are the outcomes based off your roll, in order of increasing severity
+
+		1-63: Knocked down for a few seconds and a bit of brute and stamina damage
+		64-83: Knocked silly, gain some confusion as well as the above
+		84-93: Cranial trauma, get a concussion and go deaf for a bit, plus more damage
+		94-98: Knocked unconscious, get a concussion and a random mild brain trauma, as well as medium deafness and a fair amount of damage
+		99-100: Break your spinal cord, get paralyzed, take a bunch of damage too. Very unlucky!
+
+*/
+
+/datum/component/tackler/proc/splat(mob/living/carbon/user, atom/hit, datum/thrownthing/throwingdatum)
+	var/oopsie_mod = 0
+	var/danger_zone = (speed - 1) * 15 // for every extra speed we have over 1, take away 10 of the safest chance
+	danger_zone = max(min(danger_zone, 100), 1)
+
+	if(ishuman(user))
+		var/mob/living/carbon/human/S = user
+		var/head_slot = S.get_item_by_slot(ITEM_SLOT_HEAD)
+		if(head_slot && (istype(head_slot,/obj/item/clothing/head/helmet) || istype(head_slot,/obj/item/clothing/head/hardhat)))
+			oopsie_mod -= 5
+
+	if(HAS_TRAIT(user, TRAIT_CLUMSY))
+		oopsie_mod += 6 //honk!
+
+	var/oopsie = rand(danger_zone, 100)
+
+	switch(oopsie)
+		if(99 to 100)
+			// can you imagine standing around minding your own business when all of the sudden some guy fucking launches himself into a wall at full speed and irreparably paralyzes himself?
+			user.visible_message("<span class='danger'>[user] slams face-first into [hit] at an awkward angle, severing [user.p_their()] spinal column with a sickening crack! Holy shit!</span>", "<span class='userdanger'>You slam face-first into [hit] at an awkward angle, severing your spinal column with a sickening crack! Holy shit!</span>")
+			user.take_bodypart_damage(stamina=30, brute=40)
+			playsound(user, 'sound/effects/blobattack.ogg', 60, TRUE)
+			playsound(user, 'sound/effects/splat.ogg', 70, TRUE)
+			user.emote("scream")
+			user.gain_trauma(/datum/brain_trauma/severe/paralysis/paraplegic) // oopsie indeed!
+			shake_camera(user, 7, 7)
+			user.overlay_fullscreen("flash", /obj/screen/fullscreen/flash)
+			user.clear_fullscreen("flash", 3.5)
+
+		if(94 to 98)
+			user.visible_message("<span class='danger'>[user] slams face-first into [hit] with a concerning squish, immediately going limp!</span>", "<span class='userdanger'>You slam face-first into [hit], and immediately lose consciousness!</span>")
+			user.take_bodypart_damage(stamina=30, brute=30)
+			user.Unconscious(100)
+			user.gain_trauma(/datum/brain_trauma/mild/concussion)
+			user.gain_trauma_type(BRAIN_TRAUMA_MILD)
```

Comment:
personally i'd do something like 60% concussion 40% random trauma, instead of both at once